tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -328,7 +328,7 @@ class TestMoConvertTF(CommonMOConvertTest):
         # TF2
         create_keras_model,
         create_keras_layer,
-        create_tf_function,
+        #create_tf_function,
```

Comment:
@popovaan, can you please take a look? Your test is switched off, how is it critical?
Please look at the code -may be something breaks it.

Reply:
The error message which led me to skipping it can be found in CI for commit `507a7548f664d1f29c90c1bbd707bd3fd26856d3` at https://github.com/openvinotoolkit/openvino/runs/11827363222, `openvino-lin` job.

Reply:
Continued in 106247.